fvtest/compilertest/build/platform/host/amd64-freebsd-clang.mk

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2016, 2017 IBM Corp. and others
```

Comment:
The copyright header should have the current year: `Copyright (c) 2016, 2018 IBM Corp. and others`.